scripts/find-unused-assets.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Array to hold the list of unreferenced files
+unreferenced_files=()
+
+# We look for assets in the /src and /public and then, check if they are referenced in
+# the code, typically in either /src, /public (manifest.json) or /functions
+while IFS= read -r -d $'\0' file; do
+    name="$(basename "$file")"
+    grep -rn -F -q "$name" ./src ./functions ./public
+    if [ $? -ne 0 ]; then
+      rm $file
+    fi
+done < <(find ./src ./public -type f \( -name "*.png" -o -name "*.svg" -o -name "*.jpg" \) -print0)
```

Comment:
It would be nice to pull out relevant file extensions into a variable at the top of the file, so it can be expanded without having to read too much code, eg

```
ASSET_EXTENSIONS=["*.png", "*.svg", "*.jpg"]
```

Reply:
Yeah, that's an option. Will make this code slightly less readable, but will on the other hand make it more explicit to modify parameters. 

There are few interesting gotchas here, like the fact that instead of piping `find` to `while` loop, we're doing process substitution to avoid spawning a subprocess inside `while` loop (which will make it impossible to access and modify the array).

Reply:
That said, I will send an update!